app/assets/stylesheets/src/shared/_datagrid.scss

```diff
@@ -74,3 +29,9 @@ a.table-sort {
   min-width: 140px;
   width: 140px;
 }
+
+.page-paginator {
+  &:not(&:first-child) {
+    margin-top: 16px;
+  }
+}
```

Comment:
I looked at the markup of all the grids and the top pagination is always the first child of the parent element, and the bottom pagination is always not the first (always after the table)